tests/tests_old/test_services.py

```diff
@@ -81,7 +81,7 @@
 
 NUM_DEVS_BEFORE = 3  # HGI, faked THM, faked REM
 NUM_DEVS_AFTER = 15  # proxy for success of cast_packets_to_rf()
-NUM_SVCS_AFTER = 10  # proxy for success
+NUM_SVCS_AFTER = 11  # proxy for success (includes add_command service)
```

Comment:
includes a lot more, so please take this (note) out in a follow-up PR. OK for now